### DISCUSSION TITLE
Add info to notes for LXC

### DISCUSSION THREAD

**Discussion by jdelliott** [community-scripts/ProxmoxVE#180] [Ideas]
It would be nice if there was an option during the selections phase of each script, that if set, would add pertinent info such as IP/port number, default password, etc, to the Notes for that LXC. For instance, when I set up my *arr stack using the scripts, the very last thing it tell you , for instance for Radarr, is to go to http://your.ip.right.here:7878. In order for me not to lose that info, I copy and paste it into the Notes section from the console. Having it automatically put it notes, under the "Buy Me a Coffee" link, would be very nice

**Comment by Mellowlynx** [comment]
Adding it to the notes within Proxmox, would require significant changes, this will probably not happen.
It's standardized to pull only the LXC name and add it to the rest of the basic template.
As not all apps have the accounts setup during install, we would need to put that info also somewhere else.
Furthermore, some apps have extra backends that should then also be added, but others do not. 
So this will further complicate the scripts then needed. It's easier that you add it your way to the notes section.
